types/ckeditor__ckeditor5-adapter-ckfinder/index.d.ts

```diff
@@ -1,6 +1,12 @@
-// Type definitions for @ckeditor/ckeditor5-adapter-ckfinder 27.0
+// Type definitions for @ckeditor/ckeditor5-adapter-ckfinder 27.1
 // Project: https://ckeditor.com/docs/ckeditor5/latest/api/adapter-ckfinder.html
 // Definitions by: Federico Panico <https://github.com/fedemp>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // Minimum TypeScript Version: 4.2
-export { default as UploadAdapter } from "./src/uploadadapter";
+import UploadAdapter from './src/uploadadapter';
+
+declare const _default: {
+    UploadAdapter: typeof UploadAdapter;
+};
+
+export default _default;
```

Comment:
I think this should be just:
```suggestion
export {
	UploadAdapter
}
```
https://github.com/ckeditor/ckeditor5/blob/master/packages/ckeditor5-adapter-ckfinder/src/index.js#L10